test.js

```diff
@@ -1,12 +1,8 @@
-'use strict';
-var test = require('ava');
-var runApplescript = require('./');
+import test from 'ava';
+import fn from './';
 
-test(function (t) {
-	t.plan(2);
+test(async t => {
+	const res = await fn('return "unicorn"');
```

Comment:
I'd prefer the latter as its shorter and IMHO doesn't really change the readability but it's up to @sindresorhus, WDYT?